packages/lib-user/src/components/UserHome/components/Dashboard/components/StatsTabs/StatsTabsContainer.js

```diff
@@ -35,7 +35,7 @@ export default function StatsTabsContainer({ user }) {
 
   const allTimeQuery = getDateInterval({
     endDate: todayUTC,
-    startDate: user?.created_at?.substring(0, 10) || '2015-07-01'
+    startDate: user?.created_at?.substring(0, 10) || '2015-03-17'
```

Comment:
Leave an inline comment here for why we have 2015-03-17 as the default date. Helpful for future programmers :)

Reply:
Refactored in [c0cf29a](https://github.com/zooniverse/front-end-monorepo/pull/6161/commits/c0cf29a98c40d019a9ee77903d6a66dd57b165ca)